konnectd/changelog/0.3.0_2020-04-14/client_names.md

```diff
@@ -0,0 +1,5 @@
+Change: improve client identifiers for end users
```

Comment:
Codacy found an issue: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)